### PR TITLE
Mention bumpfee in 0.14 release notes.

### PR DESCRIPTION
**Pull request by ryanofsky**

### CODE REVIEW

**Comment by sdaftuar** [comment]
ACK

File: doc/release-notes.md [hunk 1]
```diff
@@ -247,6 +248,11 @@ Low-level RPC changes
    optimizations to memory management. See [Pull #8753](https://github.com/bitcoin/bitcoin/pull/8753)
    for more information.
 
+ - A new RPC command `bumpfee` has been added which allows replacing unconfirmed
+   wallet transactions that signaled RBF (see `-walletrbf` startup option above)
```
Comment (TheBlueMatt) [inline doc/release-notes.md]: s/see/see the/

File: doc/release-notes.md [hunk 2]
```diff
@@ -247,6 +248,11 @@ Low-level RPC changes
...
+   with new transactions that pay higher fees, and should be more likely to get
```
Comment (TheBlueMatt) [inline doc/release-notes.md]: a new transaction?

File: doc/release-notes.md (hunk 1 above)
Comment (ryanofsky) [inline doc/release-notes.md]: Done in 9aaa1534d74567c1e3dbaa2efb0bba761282be0a

File: doc/release-notes.md (hunk 2 above)
Comment (ryanofsky) [inline doc/release-notes.md]: Ok, done in 9aaa1534d74567c1e3dbaa2efb0bba761282be0a.

**Review by ryanofsky — COMMENTED** [review]
Squashed 9aaa1534d74567c1e3dbaa2efb0bba761282be0a -> 29bae0c6c5a03890e3498f1ee194f5bef5426091 ([bnote.1](https://github.com/ryanofsky/bitcoin/commits/pr/bnote.1) -> [bnote.2](https://github.com/ryanofsky/bitcoin/commits/pr/bnote.2))

**Comment by petertodd** [comment]
ACK

**Comment by sipa** [comment]
ACK

**Review by gmaxwell — APPROVED** [review]
ACK.

**Review by jonasschnelli — APPROVED** [review]
ACK 29bae0c6c5a03890e3498f1ee194f5bef5426091

**Pull request merged**